build/media_source/templates/site/cassiopeia/scss/blocks/_global.scss

```
@@ -253,6 +253,11 @@ meter {
   }
 }
 
+/* margin for focus ring */
+.page-link {
+    margin: 0.25rem;;
```

```suggestion
  margin: .25rem;
```
Fix scss code style failing in drone.